examples/http-certification/assets/src/backend/Cargo.toml

```diff
@@ -1,22 +1,14 @@
 [package]
-name = "http_certification_new_assets_backend"
+name = "http_certification_assets_backend"
 version = "0.1.0"
 edition = "2021"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
-anyhow.workspace = true
 candid.workspace = true
 ic-cdk.workspace = true
-ic-cdk-macros.workspace = true
-serde.workspace = true
-serde_cbor.workspace = true
-sha2.workspace = true
-ic-certification.workspace = true
-ic-http-certification = { workspace = true, features = ["serde"] }
```

Comment:
The `serde` feature was removed because it should always be enabled in order for `add_certificate_header` to work.